lib/node_modules/@stdlib/stats/base/dists/normal/pdf/benchmark/benchmark.js

```diff
@@ -66,11 +76,15 @@ bench( pkg+':factory', function benchmark( b ) {
 	mu = 0.0;
 	sigma = 1.5;
 	mypdf = pdf.factory( mu, sigma );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -3.0, 3.0 );
+	}
 
 	b.tic();
 	for ( i = 0; i < b.iterations; i++ ) {
-		x = ( randu()*6.0 ) - 3.0;
-		y = mypdf( x );
+		y = mypdf( x[ i % 100 ] );
```

Comment:
```suggestion
		y = mypdf( x[ i % len ] );
```

Applies throughout the PR.